import-export-cli/cmd/gen.go

```diff
@@ -0,0 +1,50 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+*  WSO2 Inc. licenses this file to you under the Apache License,
+*  Version 2.0 (the "License"); you may not use this file except
+*  in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+ */
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wso2/product-apim-tooling/import-export-cli/utils"
+)
+
+// Get command related usage Info
+const GenCmdLiteral = "gen"
```

Comment:
Shall we run "go run tools/gen.go" to generate the docs (.md files) to the whole feature and commit those files here?
Also, shall we add the commands and the usage of this feature to the file in [1] as well?

[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/resources/README.html

Reply:
Added the genarated documents and added the new command details to README.html in [1]. Thanks for pointing this out.
[1] https://github.com/wso2/product-apim-tooling/blob/master/import-export-cli/resources/README.html